students/AndrewMiotke/session06/mailroom.py

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/env python3
+import os
+import sys
+import time
+import shutil
+from textwrap import dedent
+
+
+donors_list = {"Rufio": ("Rufio", [897, 200 , 200]),
```

Comment:
To make this a bit simpler, you can remove the tuple and leave just the list of donations as the values:

{"Rufio": [897, 200 , 200], ... }